tests/integration/cairo_path/contracts/regular_project/mycontract.cairo

```diff
@@ -0,0 +1,10 @@
+use external_lib::foo::foo;
```

Comment:
Can we rename this file? This is not a contract

Reply:
[rename file with sample cairo code](https://github.com/software-mansion/protostar/pull/1433/commits/2f04841d316867caeac24fe704d0eec58e1988b9)